app/javascript/components/Stats.tsx

```diff
@@ -42,17 +42,26 @@ export const Stats = ({
   }, [value]);
 
   return (
-    <section className={cx("stats", className)}>
-      <h2>
+    <section className={classNames(
+      "text-4xl leading-tight p-8 border border-border rounded inline-grid content-between gap-2 bg-background text-foreground",
+      {
+        "text-green": className?.includes("success"),
+        "text-red": className?.includes("danger"),
+        "text-orange": className?.includes("warning"),
+        "text-purple": className?.includes("info"),
+      },
+      className
+    )}>
+      <h2 className="flex gap-2 text-base leading-normal">
         {title}
         {description ? (
           <WithTooltip tip={description} position="top">
             <Icon name="info-circle" />
           </WithTooltip>
         ) : null}
       </h2>
-      <div ref={containerRef} className="overflow-hidden" style={{ overflowWrap: "initial" }}>
-        <span style={adjustedFontSize ? { fontSize: adjustedFontSize } : undefined}>{value ?? "-"}</span>
+      <div ref={containerRef} className="overflow-hidden break-words">
+        <span className={adjustedFontSize ? `text-[${adjustedFontSize}px]` : ""}>{value ?? "-"}</span>
```

Comment:
We should probably not adjust the font size at all. If we really need to we should set an inline style, Tailwind doesn't support interpolated class names